TrafficCapture/build.gradle

```diff
@@ -55,11 +55,11 @@ allprojects {
         if (project.hasProperty('excludeTests')) {
             exclude project.property('excludeTests')
         }
-        useJUnitPlatform {
-            systemProperty 'junit.jupiter.execution.parallel.enabled', 'true'
-            systemProperty 'junit.jupiter.execution.parallel.mode.default', "concurrent"
-            systemProperty 'junit.jupiter.execution.parallel.mode.classes.default', 'concurrent'
-        }
+//        useJUnitPlatform {
```

Comment:
instead of just commenting, can you set `junit.jupiter.execution.parallel.enabled = false` for individual projects with a comment

Reply:
I've uncommented these. We'll see how they do for the next couple builds.